src/middleware/dateTimeMiddleware.js

```diff
@@ -2,7 +2,7 @@ import format from 'date-fns/format'
 
 export default {
   Mutation: {
-    CreateUser: async (resolve, root, args, context, info) => {
+    Signup: async (resolve, root, args, context, info) => {
```

Comment:
please create new entry as the CreateUser is needed for seeding and admin

Reply:
in fact if we create a CreateUSer mutation in the graphpql schema, graphql will use our resolver. so maybe it a better path to change the resolve to CreateUser instead of creating a Signup.
let me know what you think.

Reply:
Yeah I think maybe it’s better to use the create user for both. We just need to keep in mind that it might be used by the admin later.